docs/source/optimizations/cq/cq_compile_qss.ipynb

```diff
@@ -21,26 +21,57 @@
     "</a>"
```

Comment:
<p>same as <code>css</code> notebook - more description / commentary would be helpful here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>